src/common.ts

```diff
@@ -28,6 +28,33 @@ export const commonState = {
   workerData: undefined
 };
 
+// True if the object implements the Transferable interface
+export function isTransferable (value : any) : boolean {
+  return value != null && typeof value === 'object' && 'transferable' in value;
+}
+
+// Internal symbol used to mark Transferable objects returned
+// by the Piscina.move() function
+const kMovable = Symbol('Piscina.kMovable');
+
+// True if object implements Transferable and has been returned
+// by the Piscina.move() function
+export function isMovable (value : any) : boolean {
+  return isTransferable(value) && value[kMovable] === true;
+}
+
+export function markMovable (value: any) : void {
+  Object.defineProperty(value, kMovable, {
+    enumerable: false,
+    value: true
+  });
```

Comment:
Is `enumerable` listed here in order to be explicit, as opposed to `writable`/`configurable`? I don’t think `configurable: true, writable: true` would be a bad choice here

Reply:
yeah, just to be explicit. And yeah, configurable/writable are fine also